### DISCUSSION TITLE
Double motion sensor configuration strange behaviour

### DISCUSSION THREAD

**Discussion by sorriso93** [RavenSystem/esp-homekit-devices#1744] [MEPLHAA Scripts]
**Detailed steps to reproduce it**
I have built a Wemos D1 Mini + DHT11 + two motion sensors.
Same configuration with one motion sensor is ok, adding the second and debugging I see a GPIO 99 not configured (was GPIO14 on json) and one motion sensor on homekit never gets triggered, nonetheless on the debug I get movement on GPIO99 that is the second sensor I bet (but not reported on GPIO14 as per the configuration)
Maybe something wrong on json config?

**JSON String**
{"c":{"n":"MOTTEMPBMAX","o":1},"a":[{"t":12,"f1":[{"g":0,"p":0}],"s":0,"i":5},{"t":12,"s":0,"i":5,"f1":[{"g":14,"p":0}]},{"t":24,"b":[],"g":2,"n":2,"z":-5,"h":10,"y0":[{"0":{"h":[{"h":"192.168.0.7","p":8086,"m":2,"u":"write?db=haadb","c":"sensor,location=bagnomax temperature=#HAA@0000,humidity=#HAA@0001"}]},"v":0,"r":1}]},{"1":{"s":[{"a":1}]},"i":0.5,"b":[],"s":0}]} 
**Used device**
Wemos D1 Mini + DHT11 + two motion sensors.
**HAA version**
10.2.0
**iOS/iPadOS version**
iOs 15.3 and 15.4
**Logs**

Jan 01 00:12:03 sensor,location=bagnomax temperature=23.4,humidity=34
Jan 01 00:12:03 <3> Net Action 192.168.0.7:8086 done
Jan 01 00:12:03 [11] Send Ev
Jan 01 00:12:03 [7] Send Ev
Jan 01 00:12:05 <1> DigI Sensor 1 GPIO 0
Jan 01 00:12:05 <1> Exec Action 1
Jan 01 00:12:05 [11] Send Ev
Jan 01 00:12:05 [7] Send Ev
Jan 01 00:12:10 <1> AutoOff
Jan 01 00:12:10 <1> DigI Sensor 0 GPIO 99
Jan 01 00:12:10 <1> Exec Action 0
Jan 01 00:12:10 [11] Send Ev
Jan 01 00:12:10 [7] Send Ev
Jan 01 00:12:10 [7] Get Ch
Jan 01 00:12:10 [7] Get Ch
Jan 01 00:12:10 [7] Get Ch
Jan 01 00:12:10 [7] Get Ch
Jan 01 00:12:12 <1> DigI Sensor 1 GPIO 0
Jan 01 00:12:12 <1> Exec Action 1
Jan 01 00:12:12 [11] Send Ev
Jan 01 00:12:12 [7] Send Ev
**Screen-shots or videos showing issue**

**Comment by sorriso93** [comment]
Update, I tried also a simple configuration, seems that GPIO14 is not good as json config.
With this configuration I don't get movement on home app for this sensor, don't know if it's related

Jan 01 00:00:00 

Jan 01 00:00:00 
Jan 01 00:00:00 Home Accessory Architect v10.2.0
Jan 01 00:00:00 (c) 2018-2021 José Antonio Jiménez Campos

Jan 01 00:00:00 
Jan 01 00:00:00 NORMAL

Jan 01 00:00:00 
Jan 01 00:00:00 JSON:
Jan 01 00:00:00  {"c":{"m":5,"n":"TEST","o":1},"a":[{"t":12,"f1":[{"g":14,"p":1}],"i":3,"s":0},{"1":{"s":[{"a":1}]},"i":0.5,"b":[],"s":0}]}

Jan 01 00:00:00 
Jan 01 00:00:00 Hostname TEST
Jan 01 00:00:00 Ping period 5 secs
Jan 01 00:00:00 Setup mode time 5 secs
Jan 01 00:00:00 HK Server Clients 20
Jan 01 00:00:00 Unsecure connections 0
Jan 01 00:00:00 mDNS TTL 4500
Jan 01 00:00:00 Toggles to enter setup 8
Jan 01 00:00:00 <0> Exec Action 0

Jan 01 00:00:00 
Jan 01 00:00:00 ** ACCESSORY 1

Jan 01 00:00:00 
Jan 01 00:00:00 * SERVICE 1
Jan 01 00:00:00 Type 12: BINARY SENSOR
Jan 01 00:00:00 KS main 1 child 1
Jan 01 00:00:00 Serial F445D9-1
Jan 01 00:00:00 Exec Ac on Boot 1
Jan 01 00:00:00 New Input GPIO 14: pullup 1, inv 0, filter 0, mode 0
Jan 01 00:00:00 New DigI GPIO 14: type 1, status 1
Jan 01 00:00:00 <1> DigI Sensor 1 GPIO 99
Jan 01 00:00:00 <1> Exec Action 1
Jan 01 00:00:00 Free Heap 41360

Jan 01 00:00:00 
Jan 01 00:00:00 ** ACCESSORY 2

Jan 01 00:00:00 
Jan 01 00:00:00 * SERVICE 2
Jan 01 00:00:00 Type 1: SWITCH / OUTLET
Jan 01 00:00:00 KS main 1 child 1
Jan 01 00:00:00 Serial F445D9-2
Jan 01 00:00:00 New System Action 1: val 1
Jan 01 00:00:00 Exec Ac on Boot 1
Jan 01 00:00:00 Init State 0
Jan 01 00:00:00 Set init state
Jan 01 00:00:00 <2> Setter ON 0
Jan 01 00:00:00 Setup mode trigger -127/8
Jan 01 00:00:00 <2> Exec Action 0
Jan 01 00:00:00 Free Heap 40848

Jan 01 00:00:00 
Jan 01 00:00:00 Device Category: 10

Jan 01 00:00:00 
Jan 01 00:00:01 Wifi config init
Jan 01 00:00:01 <0> Exec Action 1
Jan 01 00:00:01 Saved BSSID: c0c9e39cd086
Jan 01 00:00:01 Wifi Mode: Roaming
Jan 01 00:00:01 Start Wifi smart connect scan
Jan 01 00:00:01 Set hostname
Jan 01 00:00:02 Disarming Emergency Setup Mode
Jan 01 00:00:03 <1> AutoOff
Jan 01 00:00:03 <1> DigI Sensor 0 GPIO 99
Jan 01 00:00:03 <1> Exec Action 0
Jan 01 00:00:05 Saving states
Jan 01 00:00:05 scandone
Jan 01 00:00:05 Searching best BSSID for MAXNETHOME
Jan 01 00:00:05 RSSI -59, Ch 5 - c0c9e39cc9a6
Jan 01 00:00:05 RSSI -59, Ch 5 - c0c9e39cd102
Jan 01 00:00:05 RSSI -44, Ch 5 - c0c9e39cd086
Jan 01 00:00:05 Best BSSID: c0c9e39cd086
Jan 01 00:00:06 scandone
Jan 01 00:00:08 reconnect
Jan 01 00:00:09 scandone
Jan 01 00:00:10 add 0
Jan 01 00:00:10 aid 18
Jan 01 00:00:10 cnt 

Jan 01 00:00:10 
Jan 01 00:00:10 connected with MAXNETHOME, channel 5
Jan 01 00:00:10 dhcp client start...
Jan 01 00:00:12 ip:192.168.0.155,mask:255.255.255.0,gw:192.168.0.3
Jan 01 00:00:12 Free Heap 45924
Jan 01 00:00:12 Starting HKServer
Jan 01 00:00:12 Using existing HomeKit ID: F7:41:BF:AA:6C:8D
Jan 01 00:00:12 Found admin pairing with 45B645D6-3893-47DE-9C72-9AFF3B613D2C, disabling pair setup
Jan 01 00:00:12 Configuring mDNS
Jan 01 00:00:12 Free Heap 36068
Jan 01 00:00:12 >>> mDNS probing 1
Jan 01 00:00:12 mDNS announcement: Name=HAA-F445D9 md=RavenSystem HAA Kestrelpv=1.1id=F7:41:BF:AA:6C:8Dc#=1s#=1ff=0sf=0ci=10sh=i5bhyA== Port=5556 TTL=4500
Jan 01 00:00:13 <0> Exec Action 2
Jan 01 00:00:14 NTP upd (-1)
Jan 01 00:00:26 NTP upd (-1)
Jan 01 00:00:26 >>> mDNS probing 2
Jan 01 00:00:33 >>> mDNS working TTL 4500
Jan 01 00:00:38 NTP upd (-1)
Jan 01 00:00:47 [3] New 192.168.0.83:39393 (1/20) Free HEAP: 33312
Jan 01 00:00:47 [3] Verify 1/2
Jan 01 00:00:48 [3] Verify 2/2
Jan 01 00:00:48 [3] Pairing with 45B645D6-3893-47DE-9C72-9AFF3B613D2C
Jan 01 00:00:48 [3] Verify OK
Jan 01 00:00:48 [3] Get Acc
Jan 01 00:00:48 [3] Get Ch
Jan 01 00:00:48 [3] Get Ch
Jan 01 00:00:48 [3] Get Ch
Jan 01 00:00:48 [3] Get Ch
Jan 01 00:00:48 [3] Upd Ch
Jan 01 00:00:48 [3] Upd Ch
Jan 01 00:00:48 [3] Get Ch
Jan 01 00:00:50 NTP upd (-1)

**Comment by Hanimaniac** [comment]
I have exact the same problem and no idea how to change it.

**Comment by sorriso93** [comment]
It seems I solved it was an hardware problem using the wemos D1 PIR shield. If you change the pin by soldering on another pin configuration this is effective only using the board pin and not via the cable connected to the other interface shield
![IMG_2276](https://user-images.githubusercontent.com/33072571/154653854-2060d472-7fe1-4e63-be29-1d22808a05c2.jpg)
So basically speaking I had tow PIRs sending signal on same GPIO in my case
I solved it using the PIR shield configured on GPIO14 connected on the board as in the picture and then connection the other PIR via the cable (that use preconfigured GPIO as reported on wemos.cc)

UPDATE: this solve my hardware problem but not the homekit problem. Having both PIRs connected I get again a strange behaviour (one PIR get activated randomly, the second always off, see next post with debug)

**Comment by sorriso93** [comment]
I was optimistic. Now the problem remains that both PIR randomly becomes active (even with no movement)

Jan 01 00:00:02 Wifi config init
Jan 01 00:00:02 <0> Exec Action 1
Jan 01 00:00:02 Saved BSSID: c0c9e39cd086
Jan 01 00:00:02 Wifi Mode: Roaming
Jan 01 00:00:02 Start Wifi smart connect scan
Jan 01 00:00:02 Set hostname
Jan 01 00:00:02 Disarming Emergency Setup Mode
Jan 01 00:00:03 <3> Starting TH sensor
Jan 01 00:00:03 <3> TH sensor
Jan 01 00:00:03 <3> TEMP 19.6C
Jan 01 00:00:03 <3> Wilcard 0 0.00
Jan 01 00:00:03 <3> Exec Action 51
Jan 01 00:00:03 <3> HUM 40
Jan 01 00:00:05 <1> AutoOff
Jan 01 00:00:05 <1> DigI Sensor 0 GPIO 99
Jan 01 00:00:05 <1> Exec Action 0
Jan 01 00:00:05 <2> AutoOff
Jan 01 00:00:05 <2> DigI Sensor 0 GPIO 99
Jan 01 00:00:05 <2> Exec Action 0
Jan 01 00:00:05 Saving states
Jan 01 00:00:06 scandone
Jan 01 00:00:06 Searching best BSSID for MAXNETHOME
Jan 01 00:00:06 RSSI -62, Ch 5 - c0c9e39cc9a6
Jan 01 00:00:06 RSSI -42, Ch 5 - c0c9e39cd086
Jan 01 00:00:06 RSSI -67, Ch 5 - c0c9e39cd102
Jan 01 00:00:06 Best BSSID: c0c9e39cd086
Jan 01 00:00:07 scandone
Jan 01 00:00:08 <2> DigI Sensor 1 GPIO 14
Jan 01 00:00:08 <2> Exec Action 1
Jan 01 00:00:08 <1> DigI Sensor 1 GPIO 0
Jan 01 00:00:08 <1> Exec Action 1
Jan 01 00:00:09 reconnect
Jan 01 00:00:09 scandone
Jan 01 00:00:10 add 0
Jan 01 00:00:10 aid 15
Jan 01 00:00:10 cnt 

Jan 01 00:00:11 
Jan 01 00:00:11 connected with MAXNETHOME, channel 5
Jan 01 00:00:11 dhcp client start...
Jan 01 00:00:12 ip:192.168.0.204,mask:255.255.255.0,gw:192.168.0.3
Jan 01 00:00:12 Free Heap 44452
Jan 01 00:00:12 Starting HKServer
Jan 01 00:00:12 Using existing HomeKit ID: 5F:F8:56:29:7E:97
Jan 01 00:00:12 Found admin pairing with 45B645D6-3893-47DE-9C72-9AFF3B613D2C, disabling pair setup
Jan 01 00:00:12 Configuring mDNS
Jan 01 00:00:12 Free Heap 34592
Jan 01 00:00:13 <2> AutoOff
Jan 01 00:00:13 <2> DigI Sensor 0 GPIO 99
Jan 01 00:00:13 <2> Exec Action 0
Jan 01 00:00:13 <1> AutoOff
Jan 01 00:00:13 <1> DigI Sensor 0 GPIO 99
Jan 01 00:00:13 <1> Exec Action 0
Jan 01 00:00:13 >>> mDNS probing 1
Jan 01 00:00:13 mDNS announcement: Name=HAA-7EAC56 md=RavenSystem HAA Kestrelpv=1.1id=5F:F8:56:29:7E:97c#=1s#=1ff=0sf=0ci=10sh=D+09EA== Port=5556 TTL=4500
Jan 01 00:00:13 <0> Exec Action 2
Jan 01 00:00:13 [5] New 192.168.0.117:13776 (1/20) Free HEAP: 28456
Jan 01 00:00:13 [5] Verify 1/2
Jan 01 00:00:13 [6] New 192.168.0.174:45257 (2/20) Free HEAP: 25856
Jan 01 00:00:13 [6] Verify 1/2
Jan 01 00:00:14 [5] Verify 2/2
Jan 01 00:00:14 [5] Pairing with 45B645D6-3893-47DE-9C72-9AFF3B613D2C
Jan 01 00:00:14 [5] Verify OK
Jan 01 00:00:14 [6] Verify 2/2
Jan 01 00:00:14 [6] Pairing with 45B645D6-3893-47DE-9C72-9AFF3B613D2C
Jan 01 00:00:14 [6] Verify OK
Jan 01 00:00:14 [5] Get Acc
Jan 01 00:00:14 NTP upd (-1)
Jan 01 00:00:15 [6] Get Acc
Jan 01 00:00:15 [5] Upd Ch
Jan 01 00:00:15 [5] Upd Ch
Jan 01 00:00:15 [5] Upd Ch
Jan 01 00:00:15 [5] Upd Ch
Jan 01 00:00:15 [5] Get Ch
Jan 01 00:00:15 [5] Get Ch
Jan 01 00:00:15 [5] Get Ch
Jan 01 00:00:20 >>> mDNS probing 2
Jan 01 00:00:22 <2> DigI Sensor 1 GPIO 14
Jan 01 00:00:22 <2> Exec Action 1
Jan 01 00:00:22 <1> DigI Sensor 1 GPIO 0
Jan 01 00:00:22 <1> Exec Action 1
Jan 01 00:00:22 [5] Send Ev
Jan 01 00:00:26 NTP upd (-1)
Jan 01 00:00:27 >>> mDNS working TTL 4500
Jan 01 00:00:27 <2> AutoOff
Jan 01 00:00:27 <2> DigI Sensor 0 GPIO 99
Jan 01 00:00:27 <2> Exec Action 0
Jan 01 00:00:27 <1> AutoOff
Jan 01 00:00:27 <1> DigI Sensor 0 GPIO 99
Jan 01 00:00:27 <1> Exec Action 0
Jan 01 00:00:27 [5] Send Ev
Jan 01 00:00:31 <2> DigI Sensor 1 GPIO 14
Jan 01 00:00:31 <2> Exec Action 1
Jan 01 00:00:31 <1> DigI Sensor 1 GPIO 0
Jan 01 00:00:31 <1> Exec Action 1
Jan 01 00:00:31 [5] Send Ev
Jan 01 00:00:33 <3> TH sensor
Jan 01 00:00:33 <3> TEMP 19.6C
Jan 01 00:00:33 <3> HUM 40
Jan 01 00:00:36 <2> AutoOff
Jan 01 00:00:36 <2> DigI Sensor 0 GPIO 99
Jan 01 00:00:36 <2> Exec Action 0
Jan 01 00:00:36 <1> AutoOff
Jan 01 00:00:36 <1> DigI Sensor 0 GPIO 99
Jan 01 00:00:36 <1> Exec Action 0
Jan 01 00:00:36 [5] Send Ev
Jan 01 00:00:38 NTP upd (-1)
Jan 01 00:00:48 <2> DigI Sensor 1 GPIO 14
Jan 01 00:00:48 <2> Exec Action 1
Jan 01 00:00:48 <1> DigI Sensor 1 GPIO 0
Jan 01 00:00:48 <1> Exec Action 1
Jan 01 00:00:48 [5] Send Ev
Jan 01 00:00:50 NTP upd (-1)
Jan 01 00:00:53 <2> AutoOff
Jan 01 00:00:53 <2> DigI Sensor 0 GPIO 99
Jan 01 00:00:53 <2> Exec Action 0
Jan 01 00:00:53 <1> AutoOff
Jan 01 00:00:53 <1> DigI Sensor 0 GPIO 99
Jan 01 00:00:53 <1> Exec Action 0
Jan 01 00:00:53 [5] Send Ev
Jan 01 00:00:57 <2> DigI Sensor 1 GPIO 14
Jan 01 00:00:57 <2> Exec Action 1
Jan 01 00:00:57 <1> DigI Sensor 1 GPIO 0
Jan 01 00:00:57 <1> Exec Action 1
Jan 01 00:00:57 [5] Send Ev
Jan 01 00:01:02 [5] Get Ch
Jan 01 00:01:02 <2> AutoOff
Jan 01 00:01:02 <2> DigI Sensor 0 GPIO 99
Jan 01 00:01:02 <2> Exec Action 0
Jan 01 00:01:02 <1> AutoOff
Jan 01 00:01:02 <1> DigI Sensor 0 GPIO 99
Jan 01 00:01:02 <1> Exec Action 0
Jan 01 00:01:02 [5] Send Ev
Jan 01 00:01:03 <2> DigI Sensor 1 GPIO 14
Jan 01 00:01:03 <2> Exec Action 1
Jan 01 00:01:03 <1> DigI Sensor 1 GPIO 0
Jan 01 00:01:03 <1> Exec Action 1
Jan 01 00:01:03 [5] Send Ev
Jan 01 00:01:03 <3> TH sensor
Jan 01 00:01:03 <3> TEMP 19.5C
Jan 01 00:01:03 <3> Wilcard 0 0.00
Jan 01 00:01:03 <3> Exec Action 51
Jan 01 00:01:03 <3> HUM 38
Jan 01 00:01:03 <3> Network Action 192.168.0.7:8086
Jan 01 00:01:03 Wildcard val 19.5
Jan 01 00:01:03 Wildcard val 38
Jan 01 00:01:03 <3> Payload:
Jan 01 00:01:03 POST /write?db=haadb HTTP/1.1
Jan 01 00:01:03 Host: 192.168.0.7
Jan 01 00:01:03 User-Agent: HAA/10.2.0 esp8266
Jan 01 00:01:03 Connection: close
Jan 01 00:01:03 Content-type: text/html
Jan 01 00:01:03 Content-length: 53

Jan 01 00:01:03 
Jan 01 00:01:03 sensor,location=bagnomax temperature=19.5,humidity=38
Jan 01 00:01:03 <3> Net Action 192.168.0.7:8086 done
Jan 01 00:01:03 [5] Send Ev
Jan 01 00:01:08 <2> AutoOff
Jan 01 00:01:08 <2> DigI Sensor 0 GPIO 99
Jan 01 00:01:08 <2> Exec Action 0
Jan 01 00:01:08 <1> AutoOff
Jan 01 00:01:08 <1> DigI Sensor 0 GPIO 99
Jan 01 00:01:08 <1> Exec Action 0
Jan 01 00:01:08 [5] Send Ev
Jan 01 00:01:10 [5] Get Ch
Jan 01 00:01:10 [5] Get Ch
Jan 01 00:01:10 [5] Get Ch
Jan 01 00:01:13 <2> DigI Sensor 1 GPIO 14
Jan 01 00:01:13 <2> Exec Action 1
Jan 01 00:01:13 <1> DigI Sensor 1 GPIO 0
Jan 01 00:01:13 <1> Exec Action 1
Jan 01 00:01:13 [5] Send Ev
Jan 01 00:01:18 <2> AutoOff
Jan 01 00:01:18 <2> DigI Sensor 0 GPIO 99
Jan 01 00:01:18 <2> Exec Action 0
Jan 01 00:01:18 <1> AutoOff
Jan 01 00:01:18 <1> DigI Sensor 0 GPIO 99
Jan 01 00:01:18 <1> Exec Action 0
Jan 01 00:01:18 [5] Send Ev
Jan 01 00:01:23 <2> DigI Sensor 1 GPIO 14
Jan 01 00:01:23 <2> Exec Action 1
Jan 01 00:01:23 <1> DigI Sensor 1 GPIO 0
Jan 01 00:01:23 <1> Exec Action 1
Jan 01 00:01:23 [5] Send Ev
Jan 01 00:01:28 <2> AutoOff
Jan 01 00:01:28 <2> DigI Sensor 0 GPIO 99
Jan 01 00:01:28 <2> Exec Action 0
Jan 01 00:01:28 <1> AutoOff
Jan 01 00:01:28 <1> DigI Sensor 0 GPIO 99
Jan 01 00:01:28 <1> Exec Action 0
Jan 01 00:01:28 [5] Send Ev
Jan 01 00:01:33 <3> TH sensor
Jan 01 00:01:33 <3> TEMP 19.5C
Jan 01 00:01:33 <3> HUM 39
Jan 01 00:01:33 [5] Send Ev
Jan 01 00:01:39 <2> DigI Sensor 1 GPIO 14
Jan 01 00:01:39 <2> Exec Action 1
Jan 01 00:01:39 <1> DigI Sensor 1 GPIO 0
Jan 01 00:01:39 <1> Exec Action 1
Jan 01 00:01:39 [5] Send Ev
rd$ܟ<⸮l⸮|⸮d⸮c|⸮⸮⸮⸮r⸮#⸮c⸮⸮ng⸮l'o⸮⸮⸮#x⸮⸮l;l;lx⸮o⸮⸮lcg⸮|⸮⸮d⸮c⸮⸮gn⸮d⸮⸮$`⸮og⸮lorǛ⸮n⸮⸮lx⸮o⸮;⸮⸮⸮⸮⸮co⸮|⸮⸮p⸮⸮'o⸮⸮$`⸮ngl orǛ⸮n⸮⸮ s⸮⸮n⸮⸮`s$⸮b⸮o'⸮⸮⸮p<⸮#dc⸮gx`c⸮#⸮#⸮⸮gn⸮o⸮b⸮'cp?⸮gl⸮o⸮⸮g⸮n?c⸮⸮$dpc⸮⸮`⸮'⸮⸮ollg⸮⸮⸮⸮$ۘ⸮`⸮g⸮⸮'⸮ۓ⸮c⸮l$l~rl⸮c⸮ng⸮⸮$⸮ogn⸮'⸮⸮⸮g⸮⸮n$⸮b⸮o'⸮⸮l⸮gng⸮⸮`⸮⸮⸮⸮`rd⸮gn⸮⸮|⸮⸮⸮o⸮⸮bdpp_task_hdl : 3ffefef8, prio:14, stack:512
pm_task_hdl : 3ffef608, prio:1, stack:176
frc2_timer_ta

Jan 01 00:00:00 

Jan 01 00:00:00 
Jan 01 00:00:00 Home Accessory Architect v10.2.0
Jan 01 00:00:00 (c) 2018-2021 José Antonio Jiménez Campos

Jan 01 00:00:00 
Jan 01 00:00:00 NORMAL

Jan 01 00:00:00 
Jan 01 00:00:00 JSON:
Jan 01 00:00:00  {"c":{"n":"MOTTEMPBMAX","o":1},"a":[{"t":12,"f1":[{"g":0,"p":0}],"s":0,"i":5},{"t":12,"s":0,"i":5,"f1":[{"g":14,"p":0}]},{"t":24,"b":[],"g":2,"n":2,"z":-5,"h":10,"y0":[{"0":{"h":[{"h":"192.168.0.7","p":8086,"m":2,"u":"write?db=haadb","c":"sensor,location=bagnomax temperature=#HAA@0000,humidity=#HAA@0001"}]},"v":0,"r":1}]},{"1":{"s":[{"a":1}]},"i":0.5,"b":[],"s":0}]}

Jan 01 00:00:00 
Jan 01 00:00:00 Hostname MOTTEMPBMAX
Jan 01 00:00:00 Ping period 5 secs
Jan 01 00:00:00 Setup mode time 0 secs
Jan 01 00:00:00 HK Server Clients 20
Jan 01 00:00:00 Unsecure connections 0
Jan 01 00:00:00 mDNS TTL 4500
Jan 01 00:00:00 Toggles to enter setup 8
Jan 01 00:00:00 <0> Exec Action 0

Jan 01 00:00:00 
Jan 01 00:00:00 ** ACCESSORY 1

Jan 01 00:00:00 
Jan 01 00:00:00 * SERVICE 1
Jan 01 00:00:00 Type 12: BINARY SENSOR
Jan 01 00:00:00 KS main 1 child 1
Jan 01 00:00:00 Serial 7EAC56-1
Jan 01 00:00:00 Exec Ac on Boot 1
Jan 01 00:00:00 New Input GPIO 0: pullup 0, inv 0, filter 0, mode 0
Jan 01 00:00:00 New DigI GPIO 0: type 1, status 1
Jan 01 00:00:00 <1> DigI Sensor 1 GPIO 99
Jan 01 00:00:00 <1> Exec Action 1
Jan 01 00:00:00 Free Heap 39716

Jan 01 00:00:00 
Jan 01 00:00:00 ** ACCESSORY 2

Jan 01 00:00:00 
Jan 01 00:00:00 * SERVICE 2
Jan 01 00:00:00 Type 12: BINARY SENSOR
Jan 01 00:00:00 KS main 1 child 1
Jan 01 00:00:00 Serial 7EAC56-2
Jan 01 00:00:00 Exec Ac on Boot 1
Jan 01 00:00:00 New Input GPIO 14: pullup 0, inv 0, filter 0, mode 0
Jan 01 00:00:00 New DigI GPIO 14: type 1, status 1
Jan 01 00:00:00 <2> DigI Sensor 1 GPIO 99
Jan 01 00:00:00 <2> Exec Action 1
Jan 01 00:00:00 Free Heap 39140

Jan 01 00:00:00 
Jan 01 00:00:00 ** ACCESSORY 3

Jan 01 00:00:00 
Jan 01 00:00:00 * SERVICE 3
Jan 01 00:00:00 Type 24: TEMP/HUM SENSOR
Jan 01 00:00:00 Serial 7EAC56-3
Jan 01 00:00:00 New Network Action 51: 192.168.0.7:8086
Jan 01 00:00:00 Free Heap 38240

Jan 01 00:00:00 
Jan 01 00:00:00 ** ACCESSORY 4

Jan 01 00:00:00 
Jan 01 00:00:00 * SERVICE 4
Jan 01 00:00:00 Type 1: SWITCH / OUTLET
Jan 01 00:00:00 KS main 1 child 1
Jan 01 00:00:00 Serial 7EAC56-4
Jan 01 00:00:00 New System Action 1: val 1
Jan 01 00:00:00 Exec Ac on Boot 1
Jan 01 00:00:00 Init State 0
Jan 01 00:00:00 Set init state
Jan 01 00:00:00 <4> Setter ON 0
Jan 01 00:00:00 Setup mode trigger -127/8
Jan 01 00:00:00 <4> Exec Action 0
Jan 01 00:00:00 Free Heap 37736

Jan 01 00:00:00 
Jan 01 00:00:00 Device Category: 10

Jan 01 00:00:00 
Jan 01 00:00:01 Wifi config init
Jan 01 00:00:01 <0> Exec Action 1
Jan 01 00:00:01 Saved BSSID: c0c9e39cd086
Jan 01 00:00:01 Wifi Mode: Roaming
Jan 01 00:00:01 Start Wifi smart connect scan
Jan 01 00:00:01 Set hostname
Jan 01 00:00:02 Disarming Emergency Setup Mode
Jan 01 00:00:03 <3> Starting TH sensor
Jan 01 00:00:03 <3> TH sensor
Jan 01 00:00:03 <3> TEMP 19.4C
Jan 01 00:00:03 <3> Wilcard 0 0.00
Jan 01 00:00:03 <3> Exec Action 51
Jan 01 00:00:03 <3> HUM 40
Jan 01 00:00:05 <1> AutoOff
Jan 01 00:00:05 <1> DigI Sensor 0 GPIO 99
Jan 01 00:00:05 <1> Exec Action 0
Jan 01 00:00:05 <2> AutoOff
Jan 01 00:00:05 <2> DigI Sensor 0 GPIO 99
Jan 01 00:00:05 <2> Exec Action 0
Jan 01 00:00:05 Saving states
Jan 01 00:00:05 scandone
Jan 01 00:00:05 Searching best BSSID for MAXNETHOME
Jan 01 00:00:05 RSSI -47, Ch 5 - c0c9e39cd086
Jan 01 00:00:05 RSSI -69, Ch 5 - c0c9e39cd102
Jan 01 00:00:05 RSSI -81, Ch 5 - c0c9e39cc9a6
Jan 01 00:00:05 Best BSSID: c0c9e39cd086
Jan 01 00:00:06 <2> DigI Sensor 1 GPIO 14
Jan 01 00:00:06 <2> Exec Action 1
Jan 01 00:00:06 <1> DigI Sensor 1 GPIO 0
Jan 01 00:00:06 <1> Exec Action 1
Jan 01 00:00:06 scandone
Jan 01 00:00:08 reconnect
Jan 01 00:00:09 scandone
Jan 01 00:00:10 add 0
Jan 01 00:00:10 aid 15
Jan 01 00:00:10 cnt 

Jan 01 00:00:10 
Jan 01 00:00:10 connected with MAXNETHOME, channel 5
Jan 01 00:00:10 dhcp client start...
Jan 01 00:00:11 <2> AutoOff
Jan 01 00:00:11 <2> DigI Sensor 0 GPIO 99
Jan 01 00:00:11 <2> Exec Action 0
Jan 01 00:00:11 <1> AutoOff
Jan 01 00:00:11 <1> DigI Sensor 0 GPIO 99
Jan 01 00:00:11 <1> Exec Action 0
Jan 01 00:00:12 ip:192.168.0.204,mask:255.255.255.0,gw:192.168.0.3
Jan 01 00:00:12 Free Heap 44448
Jan 01 00:00:12 Starting HKServer
Jan 01 00:00:12 Using existing HomeKit ID: 5F:F8:56:29:7E:97
Jan 01 00:00:12 Found admin pairing with 45B645D6-3893-47DE-9C72-9AFF3B613D2C, disabling pair setup
Jan 01 00:00:12 Configuring mDNS
Jan 01 00:00:12 Free Heap 34712
Jan 01 00:00:13 >>> mDNS probing 1
Jan 01 00:00:13 mDNS announcement: Name=HAA-7EAC56 md=RavenSystem HAA Kestrelpv=1.1id=5F:F8:56:29:7E:97c#=1s#=1ff=0sf=0ci=10sh=D+09EA== Port=5556 TTL=4500
Jan 01 00:00:13 <0> Exec Action 2
Jan 01 00:00:14 NTP upd (-1)
Jan 01 00:00:14 <2> DigI Sensor 1 GPIO 14
Jan 01 00:00:14 <2> Exec Action 1
Jan 01 00:00:14 <1> DigI Sensor 1 GPIO 0
Jan 01 00:00:14 <1> Exec Action 1
Jan 01 00:00:18 [3] New 192.168.0.117:14800 (1/20) Free HEAP: 31824
Jan 01 00:00:18 [3] Verify 1/2
Jan 01 00:00:19 [3] Verify 2/2
Jan 01 00:00:19 [3] Pairing with 45B645D6-3893-47DE-9C72-9AFF3B613D2C
Jan 01 00:00:19 [3] Verify OK
Jan 01 00:00:19 [3] Get Acc
Jan 01 00:00:19 [3] Get Ch
Jan 01 00:00:19 [3] Get Ch
Jan 01 00:00:19 [3] Get Ch
Jan 01 00:00:19 [3] Get Ch
Jan 01 00:00:19 <2> AutoOff
Jan 01 00:00:19 <2> DigI Sensor 0 GPIO 99
Jan 01 00:00:19 <2> Exec Action 0
Jan 01 00:00:19 <1> AutoOff
Jan 01 00:00:19 <1> DigI Sensor 0 GPIO 99
Jan 01 00:00:19 <1> Exec Action 0
Jan 01 00:00:19 [3] Get Ch
Jan 01 00:00:19 [3] Upd Ch
Jan 01 00:00:19 [3] Upd Ch
Jan 01 00:00:19 [3] Upd Ch
Jan 01 00:00:19 [3] Upd Ch
Jan 01 00:00:19 [3] Get Ch
Jan 01 00:00:20 [3] Get Ch
Jan 01 00:00:20 >>> mDNS probing 2
Jan 01 00:00:20 [3] Get Ch
Jan 01 00:00:23 <2> DigI Sensor 1 GPIO 14

**Comment by sorriso93** [comment]
@RavenSystem 
Maybe the problem is that both accessories have a Digi on GPIO99? What is Digi on GPIO99, I didn't declared it in the json....

**Comment by sorriso93** [comment]
I also tried the json from [this old issue](https://github.com/RavenSystem/esp-homekit-devices/issues/856) that seems ok but in my case has the same behaviour

**Comment by RavenSystem** [comment]
GPIO99 is showed when input is triggered by internal mechanism, like an inching time, or a service notification. There is not any bug.

**Comment by RavenSystem** [comment]
Please, use Discussions section and leave Issues section only for issues. Thanks!

https://github.com/RavenSystem/esp-homekit-devices/wiki#issues

**Reply by sorriso93** [reply]
Ok for the GPIO99 I didn't know, but the issue I opened was for the strange behavior I get with the double motion sensor on different GPIOs. Even consulting old posts regarding same configuration I didn't solved the problem neither find any difference with my json. Anyway if it is not a bug I give up; I tried almost anything but I can't solve my problem. thanks
